app/assets/stylesheets/application.scss

```diff
@@ -14,13 +14,12 @@
  *= require_self
  */
  a {
-    color: blue;
-    text-decoration: none;
+    text-decoration: none;    
  }
 
- a:visited {
-    color: blue; 
- }
+a:visited {    
+    text-decoration: none;
```

Comment:
You are using tailwind, there is a class call `no-underline` that you can use instead of `text-decoration: none;`